Assets/Scripts/States/BaseState.cs

```diff
@@ -0,0 +1,22 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public abstract class BaseState 
```

Comment:
Let's rename this as well to be more specific to being the base plushie damage state